File: medtronic/src/main/java/info/nightscout/androidaps/plugins/pump/medtronic/data/dto/PumpTimeStampedRecord.java
package info.nightscout.androidaps.plugins.pump.medtronic.data.dto;

import info.nightscout.androidaps.plugins.pump.common.utils.StringUtil;

/**
 * Created by andy on 6/2/18.
 */
public class PumpTimeStampedRecord {

    protected int decimalPrecission = 2;
    public long atechDateTime;


    public long getAtechDateTime() {
        return this.atechDateTime;
    }


    public void setAtechDateTime(long atechDateTime) {
        this.atechDateTime = atechDateTime;
    }


    public String getFormattedDecimal(double value) {
        return StringUtil.getFormatedValueUS(value, this.decimalPrecission);
    }

}

File: pump/medtronic/src/main/kotlin/app/aaps/pump/medtronic/data/dto/PumpTimeStampedRecord.kt
package app.aaps.pump.medtronic.data.dto

import app.aaps.core.utils.StringUtil

/**
 * Created by andy on 6/2/18.
 */
open class PumpTimeStampedRecord(var atechDateTime: Long = 0) {

    var decimalPrecission = 2
    // var atechDateTime: Long = 0

    open fun getFormattedDecimal(value: Double): String? {
        return StringUtil.getFormattedValueUS(value, decimalPrecission)
    }
}